Description: Hopefully unbreak pybuild by properly pushing array arguments to @result array-ref.
Author: Dmitrijs Ledkovs <dmitrij.ledkov@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1214553

--- dh-python-1.20130819.orig/dh/pybuild.pm
+++ dh-python-1.20130819/dh/pybuild.pm
@@ -88,7 +88,7 @@ sub pybuild_commands {
 	my @result;
 
 	if ($ENV{'PYBUILD_INTERPRETERS'}) {
-		push @result, "pybuild --$step @options";
+		push @result, ['pybuild', "--$step", @options];
 	}
 	else {
 		# get interpreter packages from Build-Depends{,-Indep}:
